digdag-docs/src/operators/require.md

```diff
@@ -46,7 +46,7 @@
     loop>: ${moment(last_session_time).daysInMonth()}
     _do:
       require>: daily_workflow
-      session_time: ${moment(last_session_time).add(i, 'day')}
+      session_time: ${moment(last_session_time).add(i, 'day').format('YYYY-MM-DDTHH:mm:ssZ')}
   ```
```

Comment:
Have you ever tried? `format()` instead of `format('YYYY-MM-DDTHH:mm:ssZ')` ?

Reply:
not yet, let me check

Reply:
once I have tested with this code, I confirmed `format()` is enough.
```
timezone: UTC

+step1:
  echo>: ${moment(session_time).add(1, 'day').format('X')}

+step2:
  echo>: ${moment(session_time).add(1, 'day').format()}

+step3:
  echo>: ${moment(session_time).add(1, 'day').format('YYYY-MM-DDTHH:mm:ssZ')}

+step4:
  echo>: ${moment(session_time).add(1, 'day')}
```

execution log
```
2020-12-24 18:05:13 +0900 [INFO] (main): Starting a new session project id=1 workflow name=test_session_time session_time=2020-12-25T00:00:00+00:00
2020-12-24 18:05:14 +0900 [INFO] (0018@[0:default]+test_session_time+step1): echo>: 1608940800
1608940800
2020-12-24 18:05:14 +0900 [INFO] (0018@[0:default]+test_session_time+step2): echo>: 2020-12-26T00:00:00+00:00
2020-12-26T00:00:00+00:00
2020-12-24 18:05:15 +0900 [INFO] (0018@[0:default]+test_session_time+step3): echo>: 2020-12-26T00:00:00+00:00
2020-12-26T00:00:00+00:00
2020-12-24 18:05:15 +0900 [INFO] (0018@[0:default]+test_session_time+step4): echo>: "2020-12-26T00:00:00.000Z"
"2020-12-26T00:00:00.000Z"
```

Reply:
then updated with 4597b8398688bb777e69bb29378c4c554e73d8df commit